web/servlet/home/Home.html

```diff
@@ -66,6 +66,12 @@
                                 <!-- should have open/download button as appropriate -->
                                 <a class="btn btn-default" href="http://www.withrottle.com" role="button">Learn More »</a>
                             </p>
+                            <h4><a href="https://www.microsoft.com/store/apps/9NBLGGH4VF5J">Train Throttle</a></h4>
+                            <p><em>Train Throttle</em> is a native throttle for the Windows. It is optimized for tablets and mobile.</p>
```

Comment:
I think the grammar is wrong. Is this "for Windows", or "for the Windows phone"?